Makefile

```diff
@@ -19,6 +19,9 @@ clean:
 dist:
 	@go run github.com/goreleaser/goreleaser@v1.14.1 release --snapshot --rm-dist
 
+build:
```

Comment:
Unrelated to the changes? I cannot find this make build command

Reply:
Not sure what you mean 😅

Reply:
Ah, it's actually used here: https://github.com/elastic/apm-aws-lambda/blob/7b856d97ff778feea4e8d1a11ceb422a43a46b7b/Makefile#L72

Reply:
Thanks, I was blind! 🙏